scripts/copy_pkg

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+# copy_pkg
+
+# Stop on errors
+# See https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+set -Eeuo pipefail
+
+usage() {
+    echo "Usage: $0 (mrover_arm_in|mrover_arm_out|sa_arm_in|sa_arm_out)"
```

Comment:
Maybe make a slightly more descriptive usage here, when I used this at first I was confused about what in and out meant and how to actually use it, maybe just have an example usage.